core/client/src/transaction_builder/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Utility struct to build a transaction.
+
+mod transaction_builder;
```

Comment:
Why is this in client and not somewhere else?

And why is the trait not declared in this file directly? I don't see any reason for a folder + an extra file.

Reply:
Right, I fixed it.

I wasn't sure where to put it, maybe just under `/core/`?